app/src/androidTest/java/com/github/sdpcoachme/firebase/auth/MockAuthenticator.kt

```diff
@@ -19,7 +20,14 @@ class MockAuthenticator : Authenticator {
     }
 
     override fun signOut(context: Context?, onComplete: Runnable?) {
-        realAuthenticator.signOut(context, onComplete)
+        // Done since the tests for dashboard activity have no timeout
+        // capabilities to wait for the asynchronous sign-out to finish
+        // and, therefore, sometimes would fail
+        if (context is DashboardActivity) {
+            onComplete?.run()
```

Comment:
Doesn't this prevent the real authenticator to sign out the user when the activity is dashboard ? Since in this case the authenticator function isn't called.